reference/src/glossary.md

```diff
@@ -73,6 +73,13 @@ niches. For example, the "all bits uninitialized" is an invalid bit-pattern for
 `&mut T`, but this bit-pattern cannot be used by layout optimizations, and is not a
 niche.
 
+#### Zero-sized type / ZST
+
+Types with zero size are called zero-sized types, which is abbreviated as "ZST".
+When the alignment of the type is relevant, the schema "N-ZST" which stands for
+"N-aligned zero-sized type" is used. For example: "1-ZST" stands for
+"one-aligned zero-sized type", that is, a type with a size of zero and an
+alignment requirement of one.
```

Comment:
I think this should say "at most" somewhere, i.e., a 2-ZST has alignment requirement *at most* 2.

Reply:
I would expect only _exactly_ two to count as `2-ZST`.

Reply:
lol. Well good that we have this discussion then. ;)

I didn't think about which one makes mo sense, to be fair. Just went with what seemed "obvious" -- most of the time you don't care if types are actually "more relaxed" about alignment than what you think they are.

Reply:
If we're going to lose time nitpicking this, then I would prefer to drop the generalization for now (for n=1 there's no difference) and only introduce it if we ever need the notion of an N-ZST for N > 1.

Reply:
If we need to express simple requirements we could use something like >1ZST for "ZST with alignment greater than 1"

but there is indeed no current need for this particular extension of the "1ZST" name, so we could punt it to later.

Reply:
Agreed. We can restrict this to 1-ZST for now.

@gnzlbg r=me with that change.

Reply:
Done.